ecosystem/typescript/sdk/src/tests/e2e/aptos_client.test.ts

```diff
@@ -48,6 +48,14 @@ test("gets genesis resources", async () => {
   expect(accountResource).toBeDefined();
 });
 
+// test("gets object", async () => {
+//   const client = new AptosClient(NODE_URL);
```

Comment:
can we also test for invalid address error? i.e expect().toThrow()

Reply:
Now it's same with `getAccountResource` interface. The invalid account resource error is removed.